test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -466,6 +468,10 @@ def setUpClass(cls):
         cls.parse_param()
         cls.init_param()
         cls.check_users_exist()
+        fd, path = tempfile.mkstemp(prefix="new_server_output_",
+                                    suffix=".json")
+        cls.saved_file = path
+        Server().save_configuration(path, 'w')
```

Comment:
I have few questions here - 
1. The file name `setupclass_output_`, the last underscore is not needed. If I am missing something here please point me the same. 
2. Is it mandatory to do `save_configuration` before `init_servers`? I was thinking `init_server` is creating the server object and thus you can use the `save_configuration` method of the same instead of creating another `Server()` object.